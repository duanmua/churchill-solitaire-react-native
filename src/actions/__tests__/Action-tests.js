import { cardMoved, deckTapped } from '../';

it('creates a cardMoved action', () => {
  expect(cardMoved({
    endX: 20,
    endY: 100,
    cardId: 20,
    lane: 1,
    index: 2
  })).toMatchSnapshot();
});

it('creates a deckTapped action', () => {
  expect(deckTapped()).toMatchSnapshot();
});
