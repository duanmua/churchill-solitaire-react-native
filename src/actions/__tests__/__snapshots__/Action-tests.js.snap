// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates a cardMoved action 1`] = `
Object {
  "payload": Object {
    "cardId": 20,
    "endX": 20,
    "endY": 100,
    "index": 2,
    "lane": 1,
  },
  "type": "card_moved",
}
`;

exports[`creates a deckTapped action 1`] = `
Object {
  "type": "deck_tapped",
}
`;
